0-ad-history-mod-package/simulation/templates/template_unit_support_healer_greek.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Entity parent="template_unit_support_healer">
+  <Identity>
+    <History>
+      Like most other ancient civilizations, the Ancient Greeks held gods responsible for health and illness They saw diseases as punishments and believed that healing was a divine gift that required prayers and sacrifices.
+      Hippocrates of Cos was the first to question the supernatural origin of illnesses and to clearly separate religion from medicine. He focused treatment on the human behind an illness, observing and adjusting his patient's lifestyle to try to find the cause rather than just treating the symptoms.
+      The Corpus Hippocraticum is a collection of medical texts covering topics like anatomy, hygiene, and surgery, allegedly written by Hippocrates. The most famous of them all is the Hippocratic Oath, which sets rules and ethical standards for doctors to follow. A similar oath is still in use to this day.
```

Comment:
```suggestion
      The Corpus Hippocraticum is a collection of medical texts covering topics like anatomy, hygiene, and surgery, allegedly written by Hippocrates. The most famous of them all is the Hippocratic Oath, which sets rules and ethical standards for doctors to follow. A similar oath is still in use to this day. Today, we know that most of these texts, including the Hippocratic Oath, were probably written by his followers rather than Hippocrates himself.
```